Makefile

```diff
@@ -313,19 +311,21 @@ ifneq ($(wildcard $(XSMM_DIR)/lib/libxsmm.*),)
 endif
 
 # OCCA Backends
+OCCA_BACKENDS = /cpu/occa /gpu/occa /*/occa
 ifneq ($(wildcard $(OCCA_DIR)/lib/libocca.*),)
+  $(libceeds) : CPPFLAGS += -I$(OCCA_DIR)/include
   $(libceeds) : LDFLAGS += -L$(OCCA_DIR)/lib -Wl,-rpath,$(abspath $(OCCA_DIR)/lib)
   $(libceeds) : LDLIBS += -locca
-  libceed.c += $(occa.c)
-  $(occa.c:%.c=$(OBJDIR)/%.o) $(occa.c:%=%.tidy) : CPPFLAGS += -I$(OCCA_DIR)/include
-  BACKENDS += /cpu/occa /gpu/occa /omp/occa
+  $(libceeds) : LINK = $(CXX)
+  libceed.cpp += $(occa.cpp)
+  BACKENDS += $(OCCA_BACKENDS)
 endif
 
 # CUDA Backends
-CUDA_LIB_DIR := $(wildcard $(foreach d,lib lib64,$(CUDA_DIR)/$d/libcudart.${SO_EXT}))
-CUDA_LIB_DIR := $(patsubst %/,%,$(dir $(firstword $(CUDA_LIB_DIR))))
-CUDA_LIB_DIR_STUBS := $(CUDA_LIB_DIR)/stubs
-CUDA_BACKENDS = /gpu/cuda/ref /gpu/cuda/reg /gpu/cuda/shared /gpu/cuda/gen
+# CUDA_LIB_DIR := $(wildcard $(foreach d,lib lib64,$(CUDA_DIR)/$d/libcudart.${SO_EXT}))
+# CUDA_LIB_DIR := $(patsubst %/,%,$(dir $(firstword $(CUDA_LIB_DIR))))
+# CUDA_LIB_DIR_STUBS := $(CUDA_LIB_DIR)/stubs
+# CUDA_BACKENDS = /gpu/cuda/ref /gpu/cuda/reg /gpu/cuda/shared /gpu/cuda/gen
```

Comment:
Oops, I'm testing the CUDA backend on my side and this slipped through. I wasn't able to get the CUDA backend working for me by default so I had to manually enable it.

Reply:
Where is your `CUDA_DIR` and how is it laid out? We'd like to make this work automatically if possible.

It looks like @YohannDudouit removed the use of stubs in https://github.com/ceed/libceed/commit/241a4b83dc9c714eb4bcc90729a46be02322143a#diff-b67911656ef5d18c4ae36cb6741b7965L308 so that line could be dropped.

Reply:
> I see a number of `t2*-elemrestriction` failures, including these types.

Very strange, I'll look into it in more detail. Thanks!

Reply:
It's
```
/usr/local/cuda
```
which looks like the default in the Makefile

Reply:
What do you get with `make CUDA_DIR=/usr/local/cuda print-CUDA_LIB_DIR`? Where is your `libcudart.*`? (Are you on Mac or Linux?)

Reply:
I updated CUDA to version 10 and things seem to work. I'm not able to reproduce the CUDA backend not being detected now :sweat_smile: 

The CUDA location stayed the same so I'm not sure what changed. Probably user error